pkg/enforce/enforce.go

```diff
@@ -106,7 +111,7 @@ func EnforceAll(ctx context.Context, ghc ghclients.GhClientsInterface) (EnforceA
 
 		g.Go(func() error {
 
-			repos, _, err := getAppInstallationRepos(ctx, ic)
+			repos, _, err := getAppInstallationRepos(ctx, ic, iid)
```

Comment:
This check should be before the gothread within the for loop. Check if the installation is included within the allowed_org list and uninstall otherwise